docs/operations/downgrade-ondat-2.7-to-2.6.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Downgrade Ondat from 2.7.0 to 2.6.0"
+linkTitle: Downgrade Ondat from 2.7.0 to 2.6.0
+weight: 150
+---
+
+# Overview
+
+This guide will walk you through how to downgrade from Ondat v2.7.0 to v2.6.0. You can use this procedure should you decide that you need to roll back after upgrading to 2.7.0.
+
+As part of the 2.7.0 release we are implementing a new design for mapping your Kubernetes volumes to the underlying data storage containers on disk. This will make operation and debugging of any issues much easier, as from 2.7.0 the storage files on disk will have the same name (UUID) as the Physical volumes (PV) in Kubernetes.
+
+As part of the upgrade to 2.7.0, there will be a one time step to rename these blob files on disk. This will not change the data in any way, but will update the names of the files to align with the K8s naming convention.
```

Comment:
The upgrade actually renames the blob files and upgrades the deployment database. Again I'd leave it suitably vague and say something like: "as part of the upgrade to 2.7.0 there will be a one time step to upgrade the deployment blob files and their metadata to the new format".

Reply:
Updated